### ISSUE TITLE
Recently added list, isn't

### ISSUE THREAD

**Issue by tomlokhorst** [chriseidhof/pomotv#101]
The "recently added" list on the homepage (and consequently the Atom and Twitter feeds) isn't a list of recently added videos.

Instead it is a reverse list of the videos in `videos.yaml`, which is mostly chronologically sorted. Probably due to the instruction in the readme about `editions.yaml`.

I'm not entirely sure how to solve this. Perhaps we can add an instruction in the readme about always adding new videos to the bottom of `videos.yaml`?
I think it would be nice to promote newly posted videos on the homepage, even if they were recorded a while ago.

**Comment by chriseidhof** [comment]
Yeah, I think we can sort the videos by editions, then flatten it and take the 10 most recent... It'd be cleaner, I think. WDYT?

**Comment by tomlokhorst** [comment]
Wouldn't that just keep the videos in the same chronological order it is already in?

I think we need to add the concept of "order of addition to pomo.tv", I don't think that's encoded anywhere currently.

**Comment by eelco** [comment]
IMHO this is the current data model showing its weakness. In most databases having a ‘created’ and ‘updated’ field is super easy and handy. Everything but somehow adding a date to each video when it was merged feels like a hack to me. 

I still think it’s possible with just using the YAML files, although doing it manually would suck, so it would need a script that runs (just before/when) merging and adds the dates. We could use `git blame` to add the historic dates.

(As a side note, we could also save which github user added the videos and give them a small credit, some people get motivated by that ;)
